### DISCUSSION TITLE
[Feature Request] More weather widget options

### DISCUSSION THREAD

**Discussion by hiranokite** [valinet/ExplorerPatcher#1077] [General]
Is possible to have an option to enable widget on all monitors on 10 taskbar, or disable the open on hover widget on 11 taskbar?

**Comment by valinet** [comment]
Right now, as you clearly can see, it's not available. Still, as I always say, possible if one takes the time to do it.

The title is stupid if you ask me, forgive me for the language, but if the weather widget doesn't offer "options", idk what else does. I asked a million times to have titles relevant for the content.

Windows 10 style secondary taskbars do not have support for deskbands, so something would have to be hacked together. Don't fancy working on this, again a million edge cases to test, a billion hacks to put one on top of the other etc.

Windows 11 style taskbar idk, likely doable, though I don't use that and haven't looked into it.

I am tired of these endless hacks. When I will create my own taskbar, shell etc, I will do it properly. Now, I am beyond tired hacking things together. We got to more than a working point with this app, let's benefit from it and use our computers for actual work. I think the widget is more than usable as is with the Windows 10 taskbar. I suggest enjoying what's already in there. When I do that, someone else will come with some other thing that's obviously missing by then and the cycle will never end. Feature requests, as you see, are tackled only by me, and I want to work on other things as well, rather than spend all the extra time I have polishing this program, at least in its current form. I think it reached or has to reach its maturity level. There's always room for improvement, but a limited amount of time.

Thanks

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio
